src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -730,6 +730,13 @@ void Pane::_CloseChild(const bool closeFirst)
     }
 }
 
+winrt::fire_and_forget Pane::_CloseChildRoutine(const bool closeFirst)
+{
+    co_await winrt::resume_foreground(_root.Dispatcher());

```

Comment:
I'm mildly worried that this particular callback also falls under the category of things where we should grab a weak ref to `this` at the top, then get the strong ref after the `resume_foreground` call.

Reply:
Good catch, the previous lambdas were capturing by copy